docs/index.html

```diff
@@ -39,62 +47,77 @@
                 user-select: none; /* Non-prefixed version, currently
                                       supported by Chrome and Opera */
       }
+
       /* I like sort of transparent input boxes that don't look terribly different */
       input {
         border: 1px solid transparent;
         font: inherit;
       }
+
       input:focus {
-        border: 1px solid purple;
         outline: none;
       }
+
       /* The numbering isn't really important to see */
       span.number {
         color: gray;
       }
+
       /* This was too close to the left for my liking */
       .number.big {
         margin-left: 4px;
       }
+
       /* Stolen from https://codepen.io/travisw/pen/xOvJYQ */
       .card {
         background-color: #fff;
         border-radius: 2px;
-        box-shadow: -1px 2px 4px rgba(0,0,0,.25), 1px 1px 3px rgba(0,0,0,.1);
+        border: 1px solid lightgrey;
+      }
+
+      .card .btn.remove {
+        position: absolute;
+        top: 0;
+        right: 0;
+        z-index: 100;
+      }
+
+      input.constructor {
+        font-weight: 800;
+        margin-right: 15px;
       }
+
+      .type {
+        position: relative;
+      }
```

Comment:
Sorry I got uncreative and left this class (and `.constructor`) on both the `<div>` and the `<input>`, which one do you want this rule to apply to? ^^